_posts/2018-07-31-bazel-0.16.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: posts
+title: "Bazel 0.16"
+authors:
+  - cparsons
+---
+
+The Bazel team is happy to announce a new version of Bazel, [Bazel 0.16](https://github.com/bazelbuild/bazel/releases/tag/0.16.0)!
+
+We’ve updated our design process. All design documents are now published on a separate GitHub repository. They are explicitly approved via GitHub pull requests, so that you can easily track new proposals and status changes.
+
+Here are the main changes of Bazel 0.16:
+
+
+## General changes
+
+*    Bazel now allows `@` in package names. For example, `node_modules/@namespace_foo/lib_bar` (which is a common pattern for NPM) is now an allowed package name. 
+*    The `--distdir` option is no longer experimental. This option allows to specify additional directories to look for files before trying to fetch them from the network. Files from any of the distdirs are only used if a checksum for the file is specified, and both the filename and the checksum match.
```

Comment:
please link the flag to https://docs.bazel.build/versions/master/command-line-reference.html#flag--distdir